### PR TITLE
Provide more useful logs

### PR DESCRIPTION
**Pull request by pontus**

### CODE REVIEW

File: proxy.go [hunk 1]
```diff
@@ -87,16 +88,15 @@ func (p *Proxy) notAuthorized(w http.ResponseWriter, r *http.Request) {
...
+		log.Debugf("Request not authenticated (%v)", err)
 		p.notAuthorized(w, r)
 		return
 	}
 
 	log.Debug("prepend")
 	p.prependBucketToHostPath(r)
 
-	log.Debug("forward to backend")
+	log.Debug("Forwardin to backend")
```
Comment (jbygdell) [inline proxy.go:99]:
```suggestion
	log.Debug("Forwarding to backend")
```

File: userauth.go [hunk 2]
```diff
@@ -202,55 +217,63 @@ func (u *ValidateFromToken) getjwtkey(jwtpubkeypath string) error {
...
 			}
 			if info.Mode().IsRegular() {
-				log.Debug("reading file: ", filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
+				log.Debug("Reading file: ", filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
 				keyData, err := ioutil.ReadFile(filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
 				if err != nil {
-					return fmt.Errorf("token file error: %s", err)
+					return fmt.Errorf("token file error: %v", err)
 				}
 				mapkey := re.FindStringSubmatch(info.Name())[1]
```
Comment (jbygdell) [inline userauth.go:225]:
```suggestion
				mapkey := re.FindStringSubmatch(info.Name())[1]
```

This one will generate `panic: runtime error: index out of range [1] with length 0` if there is no valid public key in the path

**Review by jbygdell — APPROVED** [review]
A typo that should be fixed but that is it.

The second comment can be addressed in another PR after we decide how to deal with it.

**Review by jbygdell — DISMISSED** [review]

File: userauth.go (hunk 2 above)
Comment (pontus) [reply userauth.go]: Fixed.

**Review by jbygdell — APPROVED** [review]

**Pull request merged**